docs/index.rst

```diff
@@ -129,36 +129,40 @@ amorphous materials.
 .. skip: next
 
 .. plot::
-    :caption: An example simulating solid-state NMR spectrums of static and MAS experiments
+    :caption: Simulation of static and MAS solid-state NMR spectra
 
     from mrsimulator import Simulator, SpinSystem, Site
     from mrsimulator.method.lib import BlochDecaySpectrum
     import matplotlib.pyplot as plt
 
     # Make Site and SpinSystem objects
-    H_site = Site(isotope="1H", shielding_symmetric={"zeta": 13.89, "eta": 0.25})
-    spin_system = SpinSystem(sites=[H_site])
+    H_site = Site(isotope = "1H", shielding_symmetric = {"zeta": 13.89, "eta": 0.25})
+    spin_system = SpinSystem(sites = [H_site])
```

Comment:
@pjgrandinetti @deepanshs Have we settled on spaces or no spaces between kewords args? If so, we need to make sure the whole documentation is consistent, including here.

Reply:
i believe the plan is to stick with pep